stock_release_channel_batch_mode_commercial_partner/tests/test_channel_release_batch.py

```diff
@@ -31,6 +31,7 @@ def test_release_auto_group_commercial_partner(self):
 
     def test_release_auto_group_commercial_partner_no_next_batch(self):
         self.channel.batch_mode = "group_commercial_partner"
-        self.pickings.need_release = False  # cheat for getting the right condition
+        # cheat for getting the right condition
+        self.pickings.move_ids.need_release = False
```

Comment:
```suggestion
        action = self.channel.release_next_batch()

```

Reply:
Sadly the pickings in the channel doesn't share the same commercial partner, we need to call `release_next_batch` two times to empty the channel.
So better to keep the cheat in place, this will work every time for what we want to test here.

Reply:
ok then